consensus/common_test.go

```diff
@@ -191,17 +191,15 @@ func decideProposal(
 	round int32,
 ) (proposal *types.Proposal, block *types.Block) {
 	cs1.mtx.Lock()
-	block, blockParts := cs1.createProposalBlock()
-	validRound := cs1.ValidRound
+	block, _ = cs1.createProposalBlock()
```

Comment:
This might be out of scope, but is there a reason `createProposalBlock()` still returns 2 values?

Reply:
This will be changed in follow-up PR with block propagation where I am removing block parts usage

Reply:
Let's keep this PR straightforward

Reply:
Can you update the opening comment to better reflect what the scope of this PR is and what not? It will help reviewers understand the trajectory of the changes to come.

Reply:
I updated the opening comment to give more information about the PR, though I am not entirely sure what do you mean by scope in this context. 

By scope I see something like:
Changes are mostly within `types` package, specifically for the Proposal struct and for those that are related to spec mismatch for Header struct. Also, there are changes in the consensus package related to code using Proposal.

Is this what you mean by reflecting scope? Just making sure we are on the same page, as I can miss/not know some concepts. Also, I do not get how this can help understanding the trajectory and scope, as @adlerjohn properly guessed that his comment is out of scope, so I think there was enough info.